### DISCUSSION TITLE
Contribution permission

### DISCUSSION THREAD

**Discussion by pbrzosko** [vueuse/vueuse#2682] [Q&A] [answered]
Hi,

I wanted to make a PR for the issue I have created. But I constantly got 403 when trying to push a new branch to the repo. Should I register somewhere as a contributor or something?

**Comment by darrylnoakes** [comment, marked as answer]
To contribute to a project on GitHub that you are not a "member" of (e.g., you do not have write permission on the repo), you need to create a fork of the repo, push the branch to your fork, and then make a PR on the main repo to merge the branch on your fork into the main repo.

**Comment by pbrzosko** [comment]
Cool, thanks a lot. It is my first time doing it so I didn’t know :)
